src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/windows-linux-hybrid-cluster.mdx

```diff
@@ -28,9 +32,9 @@ Make sure you've completed the following before going further:
 
 ## Install the current chart bundle [#install-chart-bundle]
 
-The `newrelic/nri-bundle` chart is the regular one with all the latest updates for everything on Linux nodes. You can configure the `newrelic-logging` chart to install to Windows. This feature is reliable and supported in the logging chart.
+The `newrelic/nri-bundle` chart contains the latest updates for Linux-based nodes. The `newrelic-logging` chart can also be configured during this step to install to both Linux and Windows nodes. This feature for `newrelic-logging` is reliable and supported.
```

Comment:
* How does the values.yaml file relate/influence/affect the install for Helm? 
* Can customers install helm first, then set up this file after? Or is the file necessary for directing certain behaviors when customers install helm? 

Some of this context seems out of place in this step and might be better placed during the actual Helm install. If the context needs to be here, can you explain how nri-bundle relates to the key/value pairs in this yaml file? For example, are the "charts" for nri-bundle these keys:

* nri-kube-events
* newrelic-prometheus-agent
* newrelic-infrastructure
* And others?

You also mention adding values for nodeSelector here in the case of a failed installation, but in the sample Yaml, nodeSelector is already added with some values. Are there additional values they should add? what are they?

Reply:
@akristen 

I think this doc was built assuming the user has some understanding of how our nri-bundle helm chart works (hence being nested under Advanced Configuration).

> How does the values.yaml file relate/influence/affect the install with Helm?

It is the primary configuration file that drives which charts are installed and how they are configured to run.

> Can customers install via helm first, then set up this file after? Or is the file necessary for directing certain behaviors when customers install helm?

It's necessary for a successful install / config regardless of __when__ the file is used.  It stores the configuration state.

> Some of this context seems out of place in this step and might be better placed during the actual Helm install.

There are two Helm install steps for this specific setup.  That is why I added the paragraph at the beginning to explain to users what they are about to step into as it is not our normal workflow.

> If the context needs to be here, can you explain how nri-bundle relates to the key/value pairs in this yaml file? For example, are the "charts" for nri-bundle these keys:

> nri-kube-events
newrelic-prometheus-agent
newrelic-infrastructure
And others?

These are sub-charts to the nri-bundle.  They are enabled/disabled and configured from the nri-bundle `values.yaml` file.

> You also mention adding values for nodeSelector here in the case of a failed installation, but in the sample Yaml, nodeSelector is already added with some values. Are there additional values they should add? what are they?

It depends on the environment and the cluster setup.  We could add some commented out examples but it would be up to the user to decide what is ultimately installed and how it fits inside the cluster (windows vs. linux).  One thing we could do for certain is to identify which components will not run on Windows no matter what.